Makefile

```diff
@@ -170,6 +170,7 @@ modules_boundaries:
 
 develop:
 	$(PYTHON) setup.py develop $(shell $(PYTHON26) || echo --user)
+	cd html && $(PYTHON) setup.py develop $(shell $(PYTHON26) || echo --user) && cd -
```

Comment:
Nice, but think about the future. I'm fine with hardcoding the setups for now, but I don't like putting the optional plugin in the root directory. We expect to extract more plugins so it'd be natural for me to create a directory and put the `html` there.

Also please the `pystache` should be removed from `requirements` and the `html` plugin should get custom `requirements` file, treated the same way as `develop` in `requirements` section.